src/styles.css

```diff
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", "Helvetica", sans-serif;
+}
+
+body,
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name-product {
+  font-weight: 500;
+  font-style: inherit;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 auto 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
It's better to use absolute values for line-height instead of px ones. Like 130% or 1.3 
Apply all over the document